src/components/NavBar/NavBar.jsx

```diff
@@ -58,177 +48,32 @@ const StyledInputBase = styled(InputBase)(({ theme }) => ({
 
 export default function NavBar() {
 
-  const [anchorEl, setAnchorEl] = React.useState(null);
```

Comment:
I removed things that will not be used (hamburger icon, mail icon, alert icon, etc)